src/ray/common/ray_config_def.h

```diff
@@ -502,10 +502,20 @@ RAY_CONFIG(int64_t, task_events_dropped_task_attempts_gc_threshold_s, 15 * 60)
 /// workers. Events will be evicted based on a FIFO order.
 RAY_CONFIG(uint64_t, task_events_max_num_status_events_buffer_on_worker, 100 * 1000)
 
+/// Max number of task status events that will be stored to write to file
```

Comment:
write to a file -> to export (so that we can hide impl detail of export is writing to a file)